app/main.py

```diff
@@ -1,8 +1,38 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
```

Comment:
Put each argument into separate line. It will look more readable. If all arguments don't fit into 1 line, it's better to place them in separate lines